vcpkg/commands/create.md

```diff
@@ -0,0 +1,160 @@
+---
+title: vcpkg create
+description: Command line reference for the vcpkg create command. Generates a port to package a source code project.
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 08/01/2023
+ms.prod: vcpkg
+---
+
+# vcpkg create
+
+## Synopsis
+
+```no-highlight
+vcpkg create <port-name> <url-to-source> [archive-file-name]
+```
+
+## Description
+
+Generates a port to package a source code project. Requires a port name and a URL for the source code to package.
+
+This command downloads the source code from the provided URL and then creates a `portfile.cmake` file and a `vcpkg.json` file in a new port directory.
+
+The command can save the downloaded source code using a specific file name, provided as an optional argument. If the archive file name isn't specified, the command derives a file name from its URL.
+
+It's important to understand that the port created by the `vcpkg create` command serves merely as a starting point and, in most cases, further edits are necessary for a successful build.
+
+Be aware that the `vcpkg create` command doesn't verify the uniqueness of the port name within the vcpkg ports tree. The purpose of this command is to provide a foundation for port creation. If you require more guidance on adding ports to the vcpkg curated catalog, we recommend referring to one of our [tutorials](..\examples\packaging-zipfiles.md)
```

Comment:
This section is misguided -- of course this is intended to be a foundation for port creation.

We should preserve the link to the example somehow.